client/index.html

```diff
@@ -1,31 +1,93 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="styles.css">
-    <title>Document</title>
-</head>
-<body>
-    <form>
-        <input type="text" placeholder="what are you looking for?">
-        <input type="button" value="search">
-        <input type="button" value="take me straight there">
-    </form>
-    <section>
-        <ul>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-            <li></li>
-        </ul>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Google Font Icons -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="styles.css" /> 
+    <script src="./scripts.js" defer></script>
+    <!-- API -->
+    <title>Google Clone Foolgle</title>
+  </head>
+  <body>
+    <!-- Header -->
+    <header>
+        <nav class="navbar">
+            <ul>
+                <li>
+                    <a class="link" href="">About</a>
+                </li>
+                <li>
+                    <a class="link" href="">Store</a>
+                </li>
+                <li>
+                    <a class="link" href="">Gmail</a>
+                </li>
+                <li>
+                    <a class="link" href="">Images</a>
+                </li>
+                <li>
+                    <div class="circle-shadow">
+                        <a class="user-icon" href=""><span>C</span></a>
+                    </div>
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <!-- Header Ends -->
+
+    <!-- Content -->
+    <section class="content-section">
```

Comment:
"content-section" is a bit vague - for readability could be more indicative of what that content is i.e. logo/search or /hero